src/templates/prototypes/people-id.pug

```diff
@@ -3,11 +3,41 @@ block content
 	.section--primary(id="content")
 		.container
 			.block--border__bottom
-				.section--primary__container
-					+image('../../images/diane-abbott.jpg', 'Dianne Abbott')
-					.section--primary__block
-						h1 Diane Abbott
-						p #[+link('#', 'Labour')] MP for #[+link('#', 'Hackney North and Stoke Newington')]
+					+image('../../images/kenneth_clark_1464x584.jpg', 'Dianne Abbott')
```

Comment:
Not sure why this would be the case? Still sits 2 directories in while in the public folder.